go.mod

```diff
@@ -1,28 +1,31 @@
 module github.com/csi-addons/kubernetes-csi-addons
 
+replace github.com/csi-addons/kubernetes-csi-addons/apis => ./apis
+
 go 1.20
 
 require (
 	github.com/container-storage-interface/spec v1.8.0
-	github.com/csi-addons/spec v0.2.1-0.20230606140122-d20966d2e444
+	github.com/csi-addons/kubernetes-csi-addons/apis v0.0.0-00010101000000-000000000000
+	github.com/csi-addons/spec v0.2.0
```

Comment:
Please prevent this downgrade of the csi-addons/spec, recently  added features need to have an unreleased version at the moment.

Reply:
go mod tidy is bringing all these changes.